lib/streamlit/MediaFileManager.py

```diff
@@ -94,18 +99,22 @@ class MediaFileManager(object):
     def __init__(self):
         self._files = {}
         self._session_id_to_file_ids = collections.defaultdict(
-            set
-        )  # type: typing.DefaultDict[str, typing.Set[str]]
+            lambda: {}
+        )  # type: typing.DefaultDict[str, dict[str]]
 
     def _remove(self, mediafile_or_id):
         """Deletes MediaFile via file_id lookup.
```

Comment:
Actually this docstring is no longer correct. Neither is the name of this function.